app/validators/answers/option_answer_validator.py

```diff
@@ -77,3 +79,10 @@ def validate_answer_actions(self):
 
             if block_id and block_id not in self.block_ids:
                 self.add_error(self.BLOCK_ID_MISSING, block_id=block_id)
+
+    def validate_default_exists_in_options(self):
+        default_value = self.answer.get("default")
+        if default_value and default_value not in [
+            option["value"] for option in self.options
+        ]:
+            self.add_error(self.DEFAULT_MISMATCH)
```

Comment:
It would be good to make better use of the kwargs argument to add_error to provide better context to the developer. This saves a step of having to hunt it down first.

```python
self.add_error(self.DEFAULT_MISMATCH, default_value=default_value)
```

Reply:
Done!👍